src/test-error/plain-text-decorator.js

```diff
@@ -0,0 +1,27 @@
+const ACTUAL_MSG_PREFIX = 'Actual:   ';
+
+function escapeNewLines (str) {
+    return str.replace(/(\r\n|\n|\r)/gm, '\\n');
+}
+
+export default {
+    'actual': (str) => ACTUAL_MSG_PREFIX + escapeNewLines(str),
+
+    //NOTE: add gaps with length equal the length of actual prefix
+    //''    =>  'Actual:   '
+    //'^'   =>  '         ^'
+    'diff-marker': (str) => str ? ACTUAL_MSG_PREFIX.replace(/./g, ' ') + str : '',
+
+    'error-category': () => '',
+
+    'expected': (str) => 'Expected: ' + escapeNewLines(str),
+
+    'js': (str) => `${str}`,
+
+    'link': (str) => `"${str}"`,
+
+    'related-code': (str) => escapeNewLines(str),
```

Comment:
Js code is just highlighted code. Related code uses in UI for the transition to the failed step and has other look.